cmd/launch.go

```diff
@@ -86,27 +87,28 @@ func newLaunchCommand(client *client.Client) *Command {
 }
 
 func runLaunch(cmdCtx *cmdctx.CmdContext) error {
-	ctx := cmdCtx.Command.Context()
-
-	dir := cmdCtx.Config.GetString("path")
+	var (
+		importedConfig bool
+		ctx            = cmdCtx.Command.Context()
+		dir            = cmdCtx.Config.GetString("path")
+		orgSlug        = cmdCtx.Config.GetString("org")
+		appConfig      = flyctl.NewAppConfig()
+		io             = iostreams.FromContext(ctx)
```

Comment:
Does this actually work? We have a general issue that the app context is not initialized for some old commands.

Reply:
I think so? It's used in machines commands as well